From 539bf059495f30e1fc82f114dca3802c257cb6e5 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Wed, 25 Oct 2017 22:55:26 +0800
Subject: [PATCH 2680/5242] MLK-16690 ARM64: dts: freescale: imx8qm: update
 A53 cpu-freq table

commit  e26b5bd6f1205eb698685ba18fb85eca2e992564 from
https://source.codeaurora.org/external/imx/linux-imx.git

SCFW removes A53 1.26GHz cpu-freq set-point, update it
for linux kernel cpu-freq driver accordingly.

SCFW patch: (674c078 Fix CPU frequency related issues)

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 88576f7..1208a09 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -2997,7 +2997,6 @@
 	operating-points = <
 		/* kHz    uV */
 		/* voltage is maintained by SCFW, so no need here */
-		1260000	   0
 		1200000	   0
 		900000	   0
 		600000	   0
-- 
1.7.9.5

